src/lib/server/InputFilter.cpp

```diff
@@ -447,9 +457,14 @@ void InputFilter::KeystrokeAction::perform(const Event &event)
   using enum EventTypes;
```

Comment:
We could use `using` here to make the code a bit more readable.

Or we could put it at the top of the file, since `Event::EventFlags` seems to be used in many places.

Or... maybe since `EventFlags` is used outside of `Event` so commonly, should it be in the `Event` class?

Thinking out loud.

```suggestion
  using enum EventTypes;
  using Flags = Event::EventFlags;
  
```

Reply:
I am not trying to make those changes at this time just making enum classes.